.github/workflows/publish_connectors.yml

```diff
@@ -16,6 +16,16 @@ on:
         description: "Whether to publish as a pre-release (tagged for development) or as a main-release (production)"
         default: pre-release
         type: string
+      publish-java-tars:
```

Comment:
making publishing java tars optional since we don't want to publish enterprise connectors tars.